style.css

```diff
@@ -171,53 +195,50 @@ h2 {
 
 /* ----------Media Query---------- */
 
-@media screen and (min-width: 665px) and (max-width: 1120px) {
-  .App {
-    padding: 0 20px;
+@media screen and (min-width: 765px) and (max-width: 1120px) {
```

Comment:
This is a curiously specific sizing rule. Could this be a magic number?

https://css-tricks.com/magic-numbers-in-css/

What problem are you trying to resolve with this - is it possible you could resolve with one of these layout classics:

https://web.dev/articles/one-line-layouts